vertical-pod-autoscaler/pkg/updater/logic/updater_test.go

```diff
@@ -136,12 +140,76 @@ func TestVPAOff(t *testing.T) {
 	vpaLister.On("List").Return([]*vpa_types.VerticalPodAutoscaler{vpaObj}, nil).Once()
 
 	updater := &updater{
-		vpaLister:               vpaLister,
-		podLister:               podLister,
-		evictionFactory:         factory,
-		evictionRateLimiter:     rate.NewLimiter(rate.Inf, 0),
-		recommendationProcessor: &test.FakeRecommendationProcessor{},
-		selectorFetcher:         target_mock.NewMockVpaTargetSelectorFetcher(ctrl),
+		vpaLister:                    vpaLister,
+		podLister:                    podLister,
+		evictionFactory:              factory,
+		evictionRateLimiter:          rate.NewLimiter(rate.Inf, 0),
+		recommendationProcessor:      &test.FakeRecommendationProcessor{},
+		selectorFetcher:              target_mock.NewMockVpaTargetSelectorFetcher(ctrl),
+		useAdmissionControllerStatus: true,
+		statusValidator:              newFakeValidator(true),
+	}
+
+	updater.RunOnce(context.Background())
+	eviction.AssertNumberOfCalls(t, "Evict", 0)
+}
+
+// TODO(krzysied): Merge tests so there is less copy-pasted code.
```

Comment:
How about extracting test setup to a separate function for now?

Reply:
Currently `TestRunOnce_InvalidStatus` is copy-paste of `TestRunOnce` with expected eviction count equaling 0.
The only reason I want to keep it this ways is the fact that I want to change it in separate PR. I assume that I'll be changing no only this test, but also all other tests in this file.